packages/math-input/src/components/icon.tsx

```diff
@@ -5,22 +5,22 @@
 import {StyleSheet} from "aphrodite";
 import * as React from "react";
 
-import {IconTypes} from "../consts";
+import {IconType} from "../enums";
 
 import {offBlack} from "./common-style";
 import MathIcon from "./math-icon";
 import SvgIcon from "./svg-icon";
 import TextIcon from "./text-icon";
 
-import type {Icon as IconPropType} from "../types";
+import type {IconConfig} from "../types";
```

Comment:
These were a confusing set of symbols.  We had an IconTypes, Icon, Icon.  In some cases we'd import the `Icon` type as `IconType` sometimes as `IconPropType` (like here).  In the end I made it so there were no same-named types to keep things clear and avoid aliasing at the import sites.